page3.html

```diff
@@ -0,0 +1,5 @@
+html>
+<body>
+Treść strony 3
```

Comment:
we tu zmien :D

Reply:
co ty piszesz baranie ty!!!11!!~!!

Reply:
> we tu zmien :D

rel

Reply:
nie ma na mnie kozaka

Reply:
jak ja z toba kolezanko na malboraska troche coca coli